kcpp_adapters/AutoGuess.json

```diff
@@ -199,11 +199,7 @@
         "assistant_start": "ASSISTANT: ",
         "assistant_end": "</s>\n"
     }
-},
-
-
-
-{
+}, {
```

Comment:
FWIW, I intentionally put this in here to ensure people didn't just stack more adapters at the end.